src/main/java/com/blakebr0/extendedcrafting/item/ItemSingularityCustom.java

```diff
@@ -63,7 +74,10 @@ public void configure(Configuration config) {
 				+ "\n- Note: you can also specify meta for item ids, by adding them to the end of the item id."
 				+ "\n- Example: minecraft:stone:3 for a meta of 3. Make the meta 32767 for wildcard value."
 				+ "\n- 'color' the color of the singularity as a hex value. http://htmlcolorcodes.com/"
-				+ "\n- Example: 123456 would color it as whatever that color is.");
+				+ "\n- Example: 123456 would color it as whatever that color is."
+                + "\n - Use the localization key \"item.ec.singularity.<name>\" to set the name of your custom Singularity."
+			    + "\n - Example: item.ec.singularity.carrot=Carrot in your resources/extendedcrafting/lang/en_us.lang"
+			    + "\n - and item.ec.singularity.carrot=морковь in your resources/extendedcrafting/lang/ru_ru.lang");
```

Comment:
Make a note that this will require ResourceLoader or a similar mod.

Reply:
I intended to make a note about the new feature on the mod's CurseForge page once we release it but yeah I should probably add it to the config line as well.